Code/GraphMol/FileParsers/testExtendedStereoParsing.cpp

```diff
@@ -1,5 +1,5 @@
 //
-//  Copyright (C) 2017 Greg Landrum
+//  Copyright (C) 2018 Rational Discovery LLC
```

Comment:
Rational Discovery no longer exists, so it's not a good candidate for new copyright statements.
If you want to put an "RDKit-associated" company there, T5 Informatics GmbH would be a candidate.

Reply:
Used T5 Informatics GmbH